README.md

```diff
@@ -156,6 +156,12 @@ You need to quantize each of them separately like this:
 ./quantize ./models/13B/ggml-model-f16.bin.1 ./models/13B/ggml-model-q4_0.bin.1 2
 ```
 
+which can be scripted like this if you are lazy (for 65B model):
+
+```bash
+for i in models/65B/ggml-model-f16.bin*;do quantized=`echo "$i" | sed -e 's/f16/q4_0/'`; ./quantize "$i" "$quantized" 2 ;done
```

Comment:
Sed is not necessary, bash and zsh can perform pattern replacement of a variable:

```suggestion
for i in models/65B/ggml-model-f16.bin* ; do ./quantize "$i" "${i/f16/q4_0/}" 2 ;done
```

Reply:
This will generate `'models/65B/ggml-model-q4_0/.bin.2'` such paths and will fail with errors, the right command should be `for i in models/65B/ggml-model-f16.bin* ; do ./quantize "$i" "${i/f16/q4_0}" 2 ;done`

Reply:
@Player-205 right, updated the suggestion above